README.md

```diff
@@ -5,8 +5,23 @@ type system, type inference, type providers and a lot of the niceties of OCaml.
 This docker image lets you get started with F# quickly and build applications
 based on F#.
 
-The image contains the latest mono version with F#.
+The `mono` image contains the latest mono version with F#, whereas the `core`
+image contains the latest .NET Core SDK and runtime.
 
 # Dependency versions used:
 
+## Mono Image
 * Mono 4.8.0.495
+
+## .NET Core Image
+* .NET Core 1.1.1 Runtime
+* .NET Core 1.0.1 SDK
+
+# ENTRYPOINT
+
+## Mono Image
+The entry point for the `mono` image is `fsharpi` so that running the image
+will result in an F# interactive REPL.
+
+The entry point for the `core` image in `/bin/bash` as there is no interactive
```

Comment:
if .NET Core can't run `fsharpi`, is there any point in having `Dockerfile`s for it at this time? Unless we start pulling nightly SDK builds of 2.0, which I think is fine, maybe even better actually, as long as we make that clear here.